lib/matplotlib/backends/backend_qt4agg.py

```diff
@@ -72,16 +45,6 @@ class FigureCanvasQTAgg(FigureCanvasQTAggBase,
 
     """
 
-    def __init__(self, figure):
-        if DEBUG:
-            print('FigureCanvasQtAgg: ', figure)
-        FigureCanvasQT.__init__(self, figure)
```

Comment:
Ah, I was lost in the classes here.  It looks like the calls out to the PyQt base classes happen in the `FigureCanvasQt.__init__` not here.

 :sheep: